homeassistant/components/homeassistant/__init__.py

```diff
@@ -70,10 +79,13 @@
             data[ATTR_ENTITY_ID] = list(ent_ids)
 
             tasks.append(
-                hass.services.async_call(domain, service.service, data, blocking)
+                asyncio.shield(
```

Comment:
I think there is no reason to shield it because gather did not forward cancel events like wait.